Library/Homebrew/cmd/audit.rb

```diff
@@ -234,6 +234,32 @@ def audit_options
     end
   end
 
+  def audit_desc
+    # For now, only check the description when using `--strict`
+    return unless @strict
+
+    desc = formula.desc
+
+    unless desc
+      problem "All formulas should have a desc."
```

Comment:
Should be `problem "All formulae should have a desc."`.

It might be a good idea to actually use the full name here, `description`, or `have a desc (Description)` so people who haven't looked at any other formula, the example formula, documentation etc *(And you'd be surprised how many people who submit PRs this applies to)* know what the `desc` stands for.